vernac/declareDef.mli

```diff
@@ -39,23 +39,39 @@ module Hook : sig
   val call : ?hook:t -> S.t -> unit
 end
 
-module ClosedDef : sig
-  type t
-
-  (* Don't use for non-interactive proofs *)
-  val of_proof_entry
-    : uctx:UState.t
-    -> Evd.side_effects Declare.proof_entry -> t
-end
+(** Declare an interactively-defined constant *)
+val declare_entry
+  :  name:Id.t
+  -> scope:locality
+  -> kind:Decls.logical_kind
+  -> ?hook:Hook.t
+  -> ?obls:(Id.t * Constr.t) list
+  -> impargs:Impargs.manual_implicits
+  -> uctx:UState.t
+  -> Evd.side_effects Declare.proof_entry
+  -> GlobRef.t
 
+(** Declares a non-interactive constant; [body] and [types] will be
+   normalized w.r.t. the passed [evar_map] [sigma]. Universes should
+   be handled properly, including minimization and restriction. Note
+   that [sigma] is checked for unresolved evars, thus you should be
+   careful not to submit open terms or evar maps with stale,
```

Comment:
This could be read as meaning that it will anomaly on open terms, could you clarify that it's a regular error?

Reply:
Not sure what change you would like to see here; actually this doesn't always fails on open terms right? There is a call to TC resolution, tho with the failure flag on.

Reply:
Please suggest an improved comment.

Reply:
>Not sure what change you would like to see here

OK, let's move on